frameioclient/client.py

```diff
@@ -230,7 +230,7 @@ def get_comment(self, comment_id, **kwargs):
     :Args:
       comment_id (string): The comment id.
     """
-    endpoint = '/comments/{id}'.format(comment_id)
+    endpoint = '/comments/{comment_id}'.format(comment_id=comment_id)
```

Comment:
I've cleaned this up in another PR coming in soon, but I don't think we need to be this verbose.